cloud_governance/common/mails/mail_message.py

```diff
@@ -391,24 +391,46 @@ def get_data_in_html_table_format(self, resources: dict):
         :param resources:
         :return:
         """
-        html_table_format = '<table width="auto" border="2" style="border-collapse: collapse;padding: 10px;border: 2px solid black;">'
-        html_style_value = "border-collapse: collapse;padding: 10px;border: 2px solid black;"
-        html_style = f'style="{html_style_value}"'
-        html_style_color = f'style="color:red;{html_style_value}"'
-        td_left, td_right = f'<td align="left" {html_style}>', '</td>'
-        td_left_color = f'<td align="left" {html_style_color}>'
-        th_left, th_right = f'<th align="left" {html_style}>', '</th>'
-        tr_left, tr_right = f'<tr {html_style}>', '</tr>'
+        style = """
+                    <style>
+                    #customers {
+                    font-family: Verdana, Helvetica, sans-serif;
+                    border-collapse: collapse;
+                    width: 100%;
+                    }
+
+                    #customers td, #customers th {
+                    border: 2px solid #000;
+                    padding: 8px;
+                    align: left;
+                    }
+
+                    #customers tr:nth-child(even){background-color: #dddddd;}
+
+                    #customers tr:hover {background-color: #B9D9B7;}
+
+                    #customers th {
+                    padding-top: 12px;
+                    padding-bottom: 12px;
+                    text-align: left;
+                    background-color: #04AA6D;
+                    color: white;
+                    }
+                    </style>
+                """
+        html_table_format = style + '<table id="customers">'
```

Comment:
Can you use format instead of +?
html_table_format =  f'{style} <table id="customers">'

Reply:
Acknowledged. Changed and verified as suggested. Thank you!